packages/vaex-core/vaex/functions.py

```diff
@@ -934,16 +934,17 @@ def td_total_seconds(x):
     :return: an expression containing the total number of seconds in a timedelta sample.
 
     Example:
+    
     >>> import vaex
     >>> import numpy as np
     >>> delta = np.array([17658720110,   11047049384039, 40712636304958, -18161254954], dtype='timedelta64[s]')
     >>> df = vaex.from_arrays(delta=delta)
     >>> df
-      #  delta
-      0  204 days +9:12:00
-      1  1 days +6:41:10
-      2  471 days +5:03:56
-      3  -22 days +23:31:15
```

Comment:
I think this was the intended style, not a bug. Aren't docstrings the same like this throughout the repo?

Reply:
No. Current docs don't render correctly (ctrl-F for "td_total_seconds"):

https://vaex.io/docs/api.html

New docs do render correctly

https://vaex--2102.org.readthedocs.build/en/2102/api.html

Sphinx threw warning about unexpected indentation.